setup.py

```diff
@@ -22,6 +22,9 @@
         'click',
         'bs4',
         'boto3',
-        'ConfigParser'
+        'ConfigParser',
         ],
+    extra_require={
```

Comment:
What's the motivation for not simply making this a required library?

Reply:
because the library is something you have to compile and so you need all the GCC stuff and lib*-dev packages which is much harder than just simply installing okta-awscli right now. And if you don't have a yubikey at all (which most of the user will be) it is just too much hassle for too many people (in my opinion)